documentation/docs/contributing/documentation.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Documentation
+sidebar_position: 5
+title: Writing documentation
 ---
 
-Documentation
-=============
+# Documentation
```

Comment:
Why do these two disagree? If it's about writing documentation, it should be named as such, and if it's just about documentation in general - why call it "writing"?